tensorflow/compiler/xla/python/xla.cc

```diff
@@ -533,6 +537,24 @@ PYBIND11_MODULE(xla_extension, m) {
   m.def("pprof_profile_to_json", &PprofProfileToJson,
         "Decodes an uncompressed pprof Profile protocol buffer into a JSON "
         "representation");
+
+  py::class_<std::vector<ncclComm_t>, std::shared_ptr<std::vector<ncclComm_t>>> ncclComm_vec(m, "ncclComm_vec");
+  py::class_<ncclUniqueId> nccl_UniqueId(m, "ncclUniqueId");
```

Comment:
Not used anymore?

Reply:
It is used in my previous version. And I met ncclUniqueId serialization problem when sharing it among multiple ray actors. Now I could delete it.